Checkout/Source/Tokenisation/CheckoutAPIService.swift

```diff
@@ -236,9 +236,11 @@ extension CheckoutAPIService {
       requestParameters,
       responseType: SecurityCodeResponse.self,
       responseErrorType: TokenisationError.ServerError.self
-    ) { [logManager, logSecurityCodeTokenResponse] tokenResponseResult, httpURLResponse in
-      logSecurityCodeTokenResponse(tokenResponseResult, httpURLResponse)
+    ) { [weak self] tokenResponseResult, httpURLResponse in
+      guard let self else { return }
 
+      logSecurityCodeTokenResponse(tokenResponseResult: tokenResponseResult, httpURLResponse: httpURLResponse)
+      
```

Comment:
sorry about it but can you revert this favour that you made upon my request? realised that it's best to not touch it